oracle/oracle.go

```diff
@@ -786,3 +786,30 @@ func nodePoolContainsNode(s []containerengine.Node, e string) bool {
 	}
 	return false
 }
+
+func (o *oracleOps) Expand(volumeID string, newSizeInGiB uint64) (uint64, error) {
```

Comment:
First check the current size of the disk. If requested `newSizeInGiB` is less than or equal to current size then return an error.
Take a look at how are we currently doing in AWS: https://github.com/libopenstorage/cloudops/blob/master/aws/aws.go#L937

Reply:
Done